index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <body>
+    <meta charset="UTF-8">
+    <title>This is th product landing page</title>
+    <link rel="stylesheet" href="css/style.css">
+    </head>
```

Comment:
Please remember that is good to use indentation to show the tags that are child elements of other tags. In this particular case, it seems like if the <head> would not contain the <meta>, the <tittle> and the <link> tags.